src/ci/scripts/install-clang.sh

```diff
@@ -12,10 +12,19 @@ source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 LLVM_VERSION="10.0.0"
 
 if isMacOS; then
-    curl -f "${MIRRORS_BASE}/clang%2Bllvm-${LLVM_VERSION}-x86_64-apple-darwin.tar.xz" | tar xJf -
+    # If the job selects a specific Xcode version, use that instead of
+    # downloading our own version.
+    if [[ -s "${SELECT_XCODE}" ]]; then
+        sudo xcode-select -s "${SELECT_XCODE}"
```

Comment:
I'd split this in two independent environment variables, each with its own separate `if` statement: `SELECT_XCODE` to choose a custom Xcode version, and `USE_XCODE_CLANG` to use the bundled clang instead of our own. This will allow to reuse `SELECT_XCODE` in the future even in builders that can use the native clang.

Reply:
> I'd split this in two independent environment variables

In that case, do you want these split back into two separate steps and scripts?

Reply:
Possibly, yeah.